src/libstd/keyword_docs.rs

```diff
@@ -116,14 +126,21 @@ mod const_keyword { }
 /// [Reference]: ../reference/items/extern-crates.html
 mod crate_keyword { }
 
+#[doc(keyword = "else")]
+//
+/// What to do when an [`if`] condition is false.
```

Comment:
```suggestion
/// What to do when an [`if`] condition does not hold.
```
In lieu of https://github.com/rust-lang/rust/issues/53667.